src/routers/controllers/removeUserCheckDetailsController.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from "express";
+import * as constants from "../../lib/constants";
+import { getTranslationsForView } from "../../lib/utils/translationUtils";
+import { AnyRecord } from "types/utilTypes";
+import { getExtraData, setExtraData } from "../../lib/utils/sessionUtils";
+
+export const removeUserCheckDetailsControllerGet = async (req: Request, res: Response): Promise<void> => {
+    const viewData = getViewData(req);
+
+    res.render(constants.REMOVE_MEMBER_PAGE, { ...viewData });
+};
+
+const getViewData = (req: Request): AnyRecord => {
+    const translations = getTranslationsForView(req.t, constants.REMOVE_MEMBER_PAGE);
+
+    // Hardcoded data will be replaced once relevant API calls available
+    const removeUserArray = getExtraData(req.session, constants.MANAGE_USERS_MEMBERSHIP);
```

Comment:
You should not name variables in a way that make them look like function. That's very confusing. Using data type in the variable name also doesn't look good. Thus, instead of `removeUserArray` the variable could be called something like `usersToRemove`, although I believe, only one of them is for removal, so more appropriate would be `existingUsers` or simply `users` and then the one that you are going to remove would be `userToRemove`, or something like that.